wakeuplabs-io/optimism-making-impact
# Feature / Sidebar-logo

## #Ticket / Issue Tracking

[OMI-12](https://wakeuplabs.atlassian.net/browse/OMI-12)

## Description

Creates and displays the logotype that goes inside the sidebar.

## Solution Adopted

- Created `features` directory to store all `sidebar` components and logic in it.
- Created `logo` component and gave it styles for small and big screens. 
- Created a `sidebar` layout inside `/layouts`.

![Screen Shot 2024-12-20 at 16 43 12](https://github.com/user-attachments/assets/315c82d2-36f0-4ca5-8bce-9a1d97736819)


## How to Test

Run `pnpm prisma:reset:dev` (it will delete you local db, apply migrations, then seed it).
Run `pnpm studio` to check the data was created successfully.
Run `pnpm dev` on the root.
